superset-frontend/package.json

```diff
@@ -84,7 +84,7 @@
     "@superset-ui/legacy-plugin-chart-sunburst": "^0.13.5",
     "@superset-ui/legacy-plugin-chart-table": "^0.13.5",
     "@superset-ui/legacy-plugin-chart-treemap": "^0.13.3",
-    "@superset-ui/legacy-plugin-chart-world-map": "^0.13.3",
+    "@superset-ui/legacy-plugin-chart-world-map": "^0.13.6",
```

Comment:
We also need to bump `@superset-ui/plugin-chart-word-cloud` to `0.13.5`

Reply:
I went ahead and did this :)

Reply:
Hope that's not a faux pas

Reply:
Oops, I meant so say `0.13.6`